app/SendAmount.tsx

```diff
@@ -1,106 +1,91 @@
 import CustomButton from '../components/utils/CustomButton';
 import i18n from '../translationService';
 import { useAppDispatch, useAppSelector } from '../hooks';
 import {
-  saveTransaction,
   selectAmount,
-  selectChosenContact,
-  selectReason,
   setAmount,
   setReason,
   unsetChosenContact,
 } from '../store/sendFormSlice';
 import { useLocalSearchParams, useRouter } from 'expo-router';
 import { useState } from 'react';
-import { Category, Transaction } from '../types/Transaction';
-import {
-  addUserTransaction,
-  getUserIdByNumber,
-  selectUser,
-  selectUserId,
-  setIsUserLoading,
-  setUserBalance,
-} from '../store/userSlice';
-import { setIsModalVisible, setModalText } from '../store/appStateSlice';
+import { setIsModalVisible } from '../store/appStateSlice';
 import { CustomModal } from '../components/utils/CustomModal';
-import { parsePhoneNumber } from 'libphonenumber-js';
-import { Contact } from 'expo-contacts';
-import { User } from '../types/User';
+import NumberInputNumberPad, {
+  NumberLabel,
+} from '../components/form/NumberInputNumberPad';
+import { parseNumberPadInputForDeposit } from '../utils/functions';
 
 export default function SendAmount() {
   const colorScheme = useColorScheme();
   const dispatch = useAppDispatch();
-  const chosenContact = useAppSelector<Contact>(selectChosenContact);
   const amount = useAppSelector<number>(selectAmount);
-  const reason = useAppSelector<string>(selectReason);
-  const currentUser = useAppSelector<User>(selectUser);
   const router = useRouter();
   const [amountError, setAmountError] = useState<boolean>(false);
-  const [reasonError, setReasonError] = useState<boolean>(false);
-  const currentUserId = useAppSelector<string>(selectUserId);
   const params = useLocalSearchParams();
   const { isRequest } = params;
 
-  const onPressSend = async () => {
-    if (!chosenContact?.phoneNumbers) return;
+  // const onPressSend = async () => {
+  //   if (!chosenContact?.phoneNumbers) return;
 
-    // phone number for recipient in send flow and 'purchaser' (aka requestee) in request flow
-    const phoneNumber = chosenContact?.phoneNumbers[0].number;
+  //   // phone number for recipient in send flow and 'purchaser' (aka requestee) in request flow
+  //   const phoneNumber = chosenContact?.phoneNumbers[0].number;
 
-    if (!phoneNumber) return;
+  //   if (!phoneNumber) return;
 
-    if (
-      currentUser?.accountBalance <= 0 ||
-      amount > currentUser?.accountBalance
-    ) {
-      dispatch(setModalText(i18n.t('sendamount_not_enough_funds')));
-      dispatch(setIsModalVisible(true));
-      return;
-    }
-    dispatch(setIsUserLoading(true));
+  //   if (
+  //     currentUser?.accountBalance <= 0 ||
+  //     amount > currentUser?.accountBalance
+  //   ) {
+  //     dispatch(setModalText(i18n.t('sendamount_not_enough_funds')));
+  //     dispatch(setIsModalVisible(true));
+  //     return;
+  //   }
+  //   dispatch(setIsUserLoading(true));
```

Comment:
Lets not include this comments in code

Reply:
yes I just wanted to be sure that we will continue with this approach then delete all the comments and what’s not necessary